solr/core/src/java/org/apache/solr/util/TestInjection.java

```diff
@@ -337,6 +342,39 @@ public static boolean injectFailUpdateRequests() {
 
     return true;
   }
+
+  public static boolean injectLeaderTragedy(SolrCore core) {
```

Comment:
What's the point of the return value?

Reply:
I'm not sure. The other methods in the class have it, so I followed the pattern.